### ISSUE TITLE
Why use cmake to build the manual?

### ISSUE THREAD

**Issue by prjemian** [nexusformat/definitions#391]
What, again, was our motivation for using cmake to build the manual?
(#230, #276, #360, #361)

Since we've moved from Makefile to cmake, the incremental builds of the manual are not working and we have lost the sidebar on every page of the manual including search and page contents.  Since most of you do not build the manual much, you might only be affected by the loss of the search box on the web documentation.  Loss of incremental builds is a *major* problem for the manual editor.

The "bug" label is because with cmake, incremental builds of the manual are now broken.

What was improved?

The losses of moving away from the Makefile do not appear to outweigh the benefits of using cmake.

**Comment by prjemian** [comment]
BTW: The Makefile system worked for me on WIndows

**Comment by prjemian** [comment]
created branch "use_make_391" to refactor using Makefile

**Comment by prjemian** [comment]
Since we moved to cmake, a new consideration of the NXDL documentation has been added when additional files are specified in a related subdirectory.  For the NXcanSAS.nxdl.xml definition, there are additional files in subdirectory "canSAS" that must also be copied to the correct subdir in the manual.

Copying such files can be implemented by either a change in the Makefile file rules or a change in the call in the Python utility program: "nxdl2rst.py".
